### PR TITLE
Support for multiple CRs.

### PR DESCRIPTION
**Pull request by jmencak**

### CODE REVIEW

**Comment by jmencak** [comment]
@sjug PTAL

File: pkg/controller/tuned/tuned_controller.go [hunk 1]
```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
 	return nil
 }
 
+func addOwnerReference(meta *metav1.ObjectMeta, tuned *tunedv1alpha1.Tuned) []metav1.OwnerReference {
```
Comment (sjug) [inline pkg/controller/tuned/tuned_controller.go]: Your desired functionality of `addOwnerReference()` over the built-in `SetControllerReference()` is to be able to manipulate two owner references (one controller and a secondary)? I would have just wrapped `SetOwnerReferences`.

File: pkg/controller/tuned/tuned_controller.go [hunk 2]
```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
...
+	var bController bool = false
+	if tuned.Name == "default" {
+		bController = true
+	}
+
+	if meta.OwnerReferences == nil {
+		meta.OwnerReferences = []metav1.OwnerReference{}
+	} else {
+		for _, owner := range meta.OwnerReferences {
+			if owner.UID == tuned.UID {
```
Comment (sjug) [inline pkg/controller/tuned/tuned_controller.go]: Is the UID safe enough to compare versus the `GroupVersion` comparison like in `referSameObject()`?

File: pkg/controller/tuned/tuned_controller.go [hunk 3]
```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
 	return nil
 }
 
+func addOwnerReference(meta *metav1.ObjectMeta, tuned *tunedv1alpha1.Tuned) []metav1.OwnerReference {
+	var bController bool = false
```
Comment (sjug) [inline pkg/controller/tuned/tuned_controller.go]: The boolean default zero value is already false. Why the name `bController`, maybe `isController`?

File: pkg/controller/tuned/tuned_controller.go [hunk 4]
```diff
@@ -69,7 +69,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	createCustomResource(mgr)
 	if err != nil {
-		log.Printf("createCustomResource(): %v\n", err)
+		log.Printf("createCustomResource(): %v", err)
```
Comment (sjug) [inline pkg/controller/tuned/tuned_controller.go]: Were there changes to the `log.Printf` to add a new line?

File: pkg/controller/tuned/tuned_controller.go (hunk 3 above)
Comment (jmencak) [reply pkg/controller/tuned/tuned_controller.go]: Isn't explicit better than implicit?  isController sounds better, will change, thanks.  b == boolean

File: pkg/controller/tuned/tuned_controller.go (hunk 4 above)
Comment (jmencak) [reply pkg/controller/tuned/tuned_controller.go]: Changes?  Not sure it changed, but the documentation clearly states the '\n' is added if it is not present.  And it definitely makes sense for logging.

File: pkg/controller/tuned/tuned_controller.go (hunk 2 above)
Comment (jmencak) [reply pkg/controller/tuned/tuned_controller.go]: Adding to TODO to look into.

**Pull request merged**